### PR TITLE
Add basic documentation for SVIDStore plugins

### PR DESCRIPTION
**Pull request by SilvaMatteus**
Signed-off-by: Matteus Silva <silvamatteus@lsd.ufcg.edu.br>

**Pull Request check list**

- [x] Commit conforms to CONTRIBUTING.md?
- [ ] Proper tests/regressions included?
- [x] Documentation updated?

**Affected functionality**

Documentation.

**Description of change**

Add basic documentation:

- `-storeSVID` flag in when creating and updating entries related SVIDStore plugins,
- Description of SVIDStore (including the new built-in) in the Agent's docs.

**Which issue this PR fixes**

### CODE REVIEW

File: doc/spire_agent.md [hunk 1]
```diff
@@ -27,6 +28,7 @@ This document is a configuration reference for SPIRE Agent. It includes informat
 | WorkloadAttestor | [docker](/doc/plugin_agent_workloadattestor_docker.md) | A workload attestor which allows selectors based on docker constructs such `label` and `image_id`|
 | WorkloadAttestor | [k8s](/doc/plugin_agent_workloadattestor_k8s.md) | A workload attestor which allows selectors based on Kubernetes constructs such `ns` (namespace) and `sa` (service account)|
 | WorkloadAttestor | [unix](/doc/plugin_agent_workloadattestor_unix.md) | A workload attestor which generates unix-based selectors like `uid` and `gid` |
+| SVIDStore        | [aws_secretsmanager](doc/plugin_agent_svidstore_aws_secretsmanager.md) | A SVIDstore which stores secrets in the AWS secrets manager with the resulting X509-SVIDs of the entries that the agent is entitled to. |
```
Comment (amartinezfayo) [inline doc/spire_agent.md:31]:
```suggestion
| SVIDStore        | [aws_secretsmanager](doc/plugin_agent_svidstore_aws_secretsmanager.md) | An SVIDstore which stores secrets in the AWS secrets manager with the resulting X509-SVIDs of the entries that the agent is entitled to. |
```

**Review by amartinezfayo — COMMENTED** [review]
Thanks @SilvaMatteus for this!
I have only one small comment.

File: doc/spire_agent.md (hunk 1 above)
Comment (SilvaMatteus) [reply doc/spire_agent.md:31]: Thanks, @amartinezfayo. I'll fix this.

**Review by amartinezfayo — APPROVED** [review]
🎉

**Pull request merged**